packages/webawesome/src/components/input/input.ts

```diff
@@ -349,7 +349,15 @@ export default class WaInput extends WebAwesomeFormAssociatedElement {
       (typeof this.value === 'number' || (this.value && this.value.length > 0));
 
     return html`
-      <label part="form-control-label label" class="label" for="input" aria-hidden=${hasLabel ? 'false' : 'true'}>
+      <label
+        part="form-control-label label"
+        class=${classMap({
+          label: true,
+          'has-slotted': hasLabelSlot,
```

Comment:
```suggestion
          'has-slotted': hasLabel,
```

We'll want to check for label content rather than the slot. Otherwise, we miss labels set via attribute, like this:
<img width="419" height="413" alt="Image" src="https://github.com/user-attachments/assets/8f0e55b6-0afa-4678-9e73-479619176a2d" />